package.json

```diff
@@ -74,7 +74,8 @@
     "typograf": "^6.14.0",
     "ua-parser-js": "^0.7.28",
     "url": "^0.11.0",
-    "url-join": "^5.0.0"
+    "url-join": "^5.0.0",
+    "utility-types": "^3.10.0"
```

Comment:
why we need it ?

Reply:
new linter found in code usage of the lib. I think before it was drawn from sub-dependencies